CONTRIBUTING.md

```diff
@@ -1,44 +1,65 @@
 Porting NCL example scripts to Python
 =====================================
-0. Fork and clone this repository and create a conda environment containing all of the software necessary to build this examples gallery webpage: [installation instructions](INSTALLATION.md)
 
-   Note that this repository uses a "git submodule" to store its data files in a separate GitHub repository. This submodule can be checked out as part of the clone process by using the `--recursive` flag:
-```
-git clone --recursive https://github.com/NCAR/GeoCAT-examples
-```
+1. Fork the [GeoCAT-examples GitHub repository](https://github.com/NCAR/GeoCAT-examples).
```

Comment:
Can you provide a link for "Fork"?

Reply:
As far I know, the "Fork" option on GitHub doesn't have a link. Should we use a screenshot instead? I am open to other suggestions

Reply:
How about a link to GitHub's documentation:

https://help.github.com/en/github/getting-started-with-github/fork-a-repo

Reply:
👍 

Done!

Reply:
Thanks!